libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -415,7 +415,14 @@ void RC_Channel::init_aux_function(const aux_func_t ch_option, const aux_switch_
     case RELAY3:
     case RELAY4:
     case CAMERA_TRIGGER:
+    case PARACHUTE_RELEASE:
+    case RESETTOARMEDYAW:
     case DO_NOTHING:
+    case ARMDISARM:
```

Comment:
Aren't all of these handled by the vehicle code?  Parachute probably shouldn't be, but....

Reply:
agree